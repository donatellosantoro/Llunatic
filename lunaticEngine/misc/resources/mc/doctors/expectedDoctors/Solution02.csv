#doctor
npi,name,hospital,email
1112,R. Chase,Sacred Hearth,chase@sh.com
1222,C. Taub,NULL,NULL
5555,G. House,PPTH,house@ppth.com
_L144,NULL,NULL,NULL

#prescription
rxid,patient,npiref
P1,J. Doe,_L144
P2,W. Smith,5555
P3,M. White,5555

