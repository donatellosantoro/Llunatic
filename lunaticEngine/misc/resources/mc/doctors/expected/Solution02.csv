#prescription
rxid,patient,npiref
P3,M. White,5555
P1,J. Doe,_L137
P2,W. Smith,5555

#doctor
npi,name,hospital,email
1222,C. Taub,_SKm1-N1(P1;J. Doe;1222;C. Taub),_SKm1-N2(P1;J. Doe;1222;C. Taub)
_L137,_SKt0-N1(_L137),_SKt0-N2(_L137),_SKt0-N3(_L137)
1112,R. Chase,Sacred Hearth,chase@sh.com
5555,G. House,PPTH,house@ppth.com
